libraries/c_sdk/standard/https/test/unit/iot_tests_https_client.c

```diff
@@ -358,89 +359,89 @@ TEST_GROUP_RUNNER( HTTPS_Client_Unit_API )
-
-    /* IotHttpsConnectionInfo_t.userBuffer.bufferLen < connectionUserBufferMinimumSize */
-    testConnInfo.userBuffer.bufferLen = connectionUserBufferMinimumSize - 1;
-    returnCode = IotHttpsClient_Connect( &connHandle, &testConnInfo );
-    TEST_ASSERT_NULL( connHandle );
-    TEST_ASSERT_EQUAL( IOT_HTTPS_INSUFFICIENT_MEMORY, returnCode );
-    /* Restore the testConnInfo for the next test. */
-    testConnInfo.userBuffer.bufferLen = pOriginalConnInfo->userBuffer.bufferLen;
-
-    /* NULL IotHttpsConnectionInfo_t.pAddress in pConnConfig.  */
-    testConnInfo.pAddress = NULL;
-    returnCode = IotHttpsClient_Connect( &connHandle, &testConnInfo );
-    TEST_ASSERT_NULL( connHandle );
-    TEST_ASSERT_EQUAL( IOT_HTTPS_INVALID_PARAMETER, returnCode );
-    /* Restore the testConnInfo for the next test. */
-    testConnInfo.pAddress = pOriginalConnInfo->pAddress;
-
-    /* IotHttpsConnectionInfo_t.addressLen is zero. */
-    testConnInfo.addressLen = 0;
-    returnCode = IotHttpsClient_Connect( &connHandle, &testConnInfo );
-    TEST_ASSERT_NULL( connHandle );
-    TEST_ASSERT_EQUAL( IOT_HTTPS_INVALID_PARAMETER, returnCode );
-
-    /* IotHttpsConnectionInfo_t.addressLen is greater than IOT_HTTPS_MAX_HOST_NAME_LENGTH */
-    testConnInfo.addressLen = IOT_HTTPS_MAX_HOST_NAME_LENGTH + 1;
-    returnCode = IotHttpsClient_Connect( &connHandle, &testConnInfo );
-    TEST_ASSERT_NULL( connHandle );
-    TEST_ASSERT_EQUAL( IOT_HTTPS_INVALID_PARAMETER, returnCode );
-
-    /* Both IotHttpsConnectionInfo_t.addressLen and IotHttpsConnectionInfo_t.pAddress are NULL. */
-    testConnInfo.pAddress = NULL;
-    returnCode = IotHttpsClient_Connect( &connHandle, &testConnInfo );
-    TEST_ASSERT_NULL( connHandle );
-    TEST_ASSERT_EQUAL( IOT_HTTPS_INVALID_PARAMETER, returnCode );
-    /* Restore the testConnInfo for the next test. */
-    testConnInfo.pAddress = pOriginalConnInfo->pAddress;
-    testConnInfo.addressLen = pOriginalConnInfo->addressLen;
-
-    /* IotHttpsConnectionInfo_t.alpnProtocolsLen is greater than IOT_HTTPS_MAX_ALPN_PROTOCOLS_LENGTH. */
-    testConnInfo.alpnProtocolsLen = IOT_HTTPS_MAX_ALPN_PROTOCOLS_LENGTH + 1;
-    returnCode = IotHttpsClient_Connect( &connHandle, &testConnInfo );
-    TEST_ASSERT_NULL( connHandle );
-    TEST_ASSERT_EQUAL( IOT_HTTPS_INVALID_PARAMETER, returnCode );
-    testConnInfo.alpnProtocolsLen = pOriginalConnInfo->alpnProtocolsLen;
-
-    /* Unknown IotHttpsConnectionInfo_t.pFlags are ignored, so that is not tested. */
-
-    /* NULL certificates will result in a network error in the underlying network stack, but is allowed in the API so '
-     * that is not tested here. */
-
-    /* IotHttpsConnectionInfo_t.pNetworkInterface is NULL. */
-    testConnInfo.pNetworkInterface = NULL;
-    returnCode = IotHttpsClient_Connect( &connHandle, &testConnInfo );
-    TEST_ASSERT_NULL( connHandle );
-    TEST_ASSERT_EQUAL( IOT_HTTPS_INVALID_PARAMETER, returnCode );
+IotHttpsReturnCode_t returnCode = IOT_HTTPS_OK;
```

Comment:
This is incrustification is different and weird, but I mean if it passes the CI then ok... lol

Reply:
Accidental formatting changes from the FreeRTOS+TCP `uncrustify.cfg` file. Reverted